.gitignore

```diff
@@ -1,3 +1,4 @@
 /target
 /assets
 /static
+.DS_Store
```

Comment:
I believe that a project's gitignore file should only list project-specific things and all globally ignored files should be placed in a global gitignore file - https://stackoverflow.com/questions/18393498/gitignore-all-the-ds-store-files-in-every-folder-and-subfolder

Reply:
I think adding it would serve more good than harm. I've always added `.DS_Store` to project repos to help make developability more seamless given there are so many developers on macOS, as most user's haven't had time to configure their global gitignore (and some aren't even aware of it).

Reply:
@ozkriff I understand your argument, but imo there is different purpose of both of those files:
- local global `.gitignore` prevents **me** from committing `.DS_Store`
- project-specific `.DS_Store` prevents anyone from accidentally committing `.DS_Store`

I believe the later solution is a desired one. Otherwise a reviewer of the pull request has to always check whether `.DS_Store` was not accidentally committed.

btw. that's rustc's `.gitignore`

https://github.com/rust-lang/rust/blob/d85e866c0d28fce32856d200fd534ac1c2c721c8/.gitignore#L3